contents/docs/integrate/_snippets/install-react-native.mdx

```diff
@@ -46,42 +46,32 @@ const MyComponent = () => {
     const posthog = usePostHog()
 
     useEffect(() => {
-        posthog?.capture("mycomponent_loaded", { foo: "bar" })
+        posthog.capture("mycomponent_loaded", { foo: "bar" })
```

Comment:
`usePostHog()` does not return undefined anymore, right?

Reply:
Indeed.